### ISSUE TITLE
Dead or alive ??? come on anyone???

### ISSUE THREAD

**Issue by tomhs** [boxeehacks/boxeehack#568]

**Comment by SendoX** [comment]
the boxee box is dead apparently the hack also.

**Comment by Kepners** [comment]
Yup. Dead. I guess the dude got fed up of all the hassle of when it went wrong.... atleast its at a working standard.

**Comment by tripflex** [comment]
Yeah about as far as you're gonna get unless someone has time and an old boxee.  Unfortunately I don't have time to mess with it and it would just be cheaper for me to use a laptop with plex and curse Boxee until the day i die :)

**Comment by StephenOTT** [comment]
Anyone interested in taking ownership?

**Comment by nielsz** [comment]
I'm sure there are some people wanting to add some functionality to BoxeeHacks.
I've tried to compile, but didn't make it: https://github.com/boxeehacks/boxeehack/issues/546

**Comment by SiqueiraBr** [comment]
hi everyone
I won a gift of a BoxeeBox and simply do not know what to do with it hahahahahaha whole platform multimedia my house is based on XBMC and tried to turn the bb on NAS, then tried to turn it into an FTP server. but the hacks are not working anymore. I really wanted to create a community that really have a commitment to work on the boxeebox because its hardware seems to be very good! Someone wants to embrace the cause? :)

**Comment by tiwilliam** [comment]
Further development can be made here:
https://github.com/cigamit/boxeehack
http://boxeed.in/

**Comment by nielsz** [comment]
Nice!

**Comment by tomhs** [comment]
Awesome thanks tiw
